src/components/dashboard/Dashboard.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import MainPanel from '../sharedComponents/MainPanel';
+import pagination from '../../util/pagination';
+
+export default class Dashboard extends Component {
+  constructor() {
+    super();
+    this.state = {
+      allTrips: [],
+      displayedTrips: [],
+    };
+    this.filter = this.filter.bind(this);
+  }
+
+  componentDidMount() {
+    setTimeout(() => {
+      this.setState({ allTrips: [{ status: 'rejected', id: 1 }, { status: 'accepted', id: 2 }, { status: 'open', id: 3 }, { status: 'accepted', id: 4 }, { status: 'accepted', id: 5 }, { status: 'rejected', id: 6 }, { status: 'accepted', id: 7 }, { status: 'open', id: 8 }, { status: 'accepted', id: 9 }, { status: 'accepted', id: 10 }, { status: 'rejected', id: 11 }, { status: 'accepted', id: 12 }, { status: 'open', id: 13 }, { status: 'accepted', id: 14 }, { status: 'accepted', id: 15 }, { status: 'rejected', id: 16 }, { status: 'accepted', id: 17 }, { status: 'open', id: 18 }, { status: 'accepted', id: 19 }, { status: 'accepted', id: 20 }] });
+      this.setState((prevState) => ({ displayedTrips: pagination(prevState.allTrips, 1) }));
+    }, 1000);
+  }
+
+  filter(newArray) {
+    this.setState({
+      displayedTrips: newArray,
+    });
+  }
+
+  render() {
+    const {
+      allTrips, displayedTrips,
+    } = this.state;
+    const requests = displayedTrips.map((el) => (
+      <p key={el.id}>
+        {el.status}
+        {' '}
+        request
+        {' '}
+        {el.id}
```

Comment:
This {`${el.status} request ${el.id}`} can be cleaner

Reply:
We deleted this dashboard as we were using it as dummy component, we are using the real request table as it was merged